server/src/main/java/org/apache/druid/server/coordination/SegmentLoadDropHandler.java

```diff
@@ -386,5 +449,22 @@ public boolean cancel(boolean interruptIfRunning)
       return true;
     }
   }
+
+  private ExecutorService getExecutorService(SegmentLoadingMode loadingMode)
+  {
+    switch (loadingMode) {
+      case TURBO:
+        return turboExec;
+      case NORMAL:
+        return standardExec;
+      default:
+        throw DruidException.defensive("Unknown execution mode [%s]", loadingMode);
+    }
```

Comment:
This can be simplified:

```suggestion
    return loadingMode == TURBO ? turboExec : normalExec;
```